### ISSUE TITLE
Verification page problems

### ISSUE THREAD

**Issue by davidnicholls125** [johnvanbreda/naturespot-d8#169]
The RAG icons are coming and going on the verification page. They were present earlier this morning but not showing now.....
Also, I've just tried to query a record and got this error popup:
![image](https://github.com/johnvanbreda/naturespot-d8/assets/80914584/7e8f533f-2187-4149-b5f4-8193f0de461f)

**Comment by davidnicholls125** [comment]
Also the redet function has stopped working - it won't bring up any species in the drop-down menu.....

**Comment by johnvanbreda** [comment]
When I try to send a query email, the browser is sending a request to the following URL: https://www.naturespot.org.uk/iform/esproxy/verificationQueryEmail/234483. This seems to be being blocked at the server level as we are getting an Access Denied 403 response which is of the style that we've seen before when it has been caused by the server (rather than Drupal). So please can you ask Clook to look into this?

I think there has been a change to caching behaviour in a recent Drupal update - the species rag request being sent to the server was caching any empty request (which happens if you try to load an empty grid), then it kept supplying the empty response even if not empty. A bit odd, but I've disabled the cache on this request and I think it's fixed.

At the moment I can't reproduce the problem with species searches - let me know if it happens again.

**Comment by davidnicholls125** [comment]
Thanks. I've raised a ticket with Clook so I'll let you know!

The redetting issue remains the same - when I type into the species box, nothing happens....
![image](https://github.com/johnvanbreda/naturespot-d8/assets/80914584/d01780a4-8f3e-4656-bf18-c09ef0351fa0)

**Comment by davidnicholls125** [comment]
Clook have clearly made some amends as it is working now (they didn't say what they have done).
The redet species list is also appearing now.....

**Comment by davidnicholls125** [comment]
The redet species menu has stopped working again this morning.....
![image](https://github.com/johnvanbreda/naturespot-d8/assets/80914584/f4cb09a2-4a11-4cae-b261-a139719b8aa1)

**Comment by davidnicholls125** [comment]
and now its working.....

**Comment by johnvanbreda** [comment]
When you find that it is not working, is that normally just after loading the page, or is it only after you've been using the verification tool for a bit? 

If it stops working, if you reload the page and immediately select the first record in the grid, then click the Redet button and start searching in the search box, does it work? (no need to actually save the redet of course). 

Could you try using the browser developer tools - see https://torquemag.io/2020/06/browser-developer-tools-tutorial/. If you leave these running, then when it fails there are a couple of things to try:
1) Check the Console tab for errors.
2) On the Network tab, click the Clear button (a circle "no entry" sign, or a trash can depending on the browser). Now try searching, then look at the requests it generated in the network tab. Click on the last request. Is there any information on the Response tab, or any other clue?
3) As above, but on the Network tab, you can normally download the requests as a HAR file and send it to me.

**Comment by davidnicholls125** [comment]
In all cases, I had been using the verification page for a while. I did refresh the page but it didn't change.
I've just tried opening it and going straight to redet and whilst it worked OK, when I pressed cancel I then got this where the redet dropdown menu continued to display on the main page.... though it went with a refresh.
I'll try the developer tools if/when it happens again.
![image](https://github.com/johnvanbreda/naturespot-d8/assets/80914584/ba3d36b9-8d77-485a-999c-516d8c92b291)
